ethereum/statetransition/src/main/java/tech/pegasys/teku/statetransition/forkchoice/ForkChoice.java

```diff
@@ -927,4 +931,25 @@ private Optional<Boolean> getOptimisticSyncing() {
   public interface OptimisticHeadSubscriber {
     void onOptimisticHeadChanged(boolean isHeadOptimistic);
   }
+
+  // Implements `validate_inclusion_lists` added in EIP-7805 - consensus/fork-choice
+  public void validateInclusionLists(
+      final UpdatableStore store,
+      final List<Transaction> inclusionListTransactions,
+      final ExecutionPayload executionPayload) {
+    final SpecConfigEip7805 eip7805 = spec.getGenesisSpec().getConfig().toVersionEip7805().get();
```

Comment:
I'd rather call it specConfigEip7805...
Not sure if the genesis spec config will work here, maybe better get the current slot from the `recentChainData` and get the spec config based on that. I'd suggest using `orElseThrow()` or `SpecConfigEipr7805::required` instead of `get`

Reply:
Or better call `spec.getCurrentSlot(recentChainData.getStore())` to get the current slot...